Modernize travis-ci config

dfreedm opened the pull request:
- polymer-cli
- docker instances for faster booting
- chrome addon instead of apt

dfreedm committed:
Modernize travis-ci config

- polymer-cli
- docker instances for faster booting
- chrome addon instead of apt

rictic commented on cleanup-passes/travis.ts:
```diff
@@ -18,25 +18,24 @@ import * as fs from 'fs';
 interface TravisConfig {
   before_script?: string[];
   addons?: {
     firefox?: string | number,
```
Prefer semicolons in interfaces

rictic commented on cleanup-passes/travis.ts:
```diff
@@ -75,7 +64,8 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
   travis.dist = 'trusty';
-  travis.sudo = 'required';
+  // use docker
+  travis.sudo = 'false';
```
This can just be left off then, yeah?

dfreedm replied:
No it has to be explicitly set to `'false'` to use docker instances.

rictic commented on cleanup-passes/travis.ts:
```diff
@@ -18,25 +18,24 @@ import * as fs from 'fs';
-import {existsSync, makeCommit} from './util';
+import { register } from '../cleanup-pass';
+import { ElementRepo } from '../element-repo';
+import { existsSync, makeCommit } from './util';
```
`npm run format` if that command exists plz

otherwise don't worry about it

dfreedm replied:
Ah this was VS Code's doing.

dfreedm replied:
No `format` script. Should I port over the `clang-format` setup from `polymer-cli`?

dfreedm committed:
use semicolons for interface

dfreedm committed:
even more semicolons in interfaces

dfreedm committed:
Merge branch 'master' into travis-modernize

dfreedm committed:
Formatting

- Remove spaces around braces
- Put C11 apt stuff in an if to continue transpiling
